bin/cache.ml

```diff
@@ -21,16 +21,27 @@ let trim =
     Cmd.info "trim" ~doc ~man
   in
   Cmd.v info
-  @@ let+ trimmed_size =
+  @@ let+ size =
```

Comment:
I've swapped these round since size comes before --trimed size in the doc.

Reply:
It's always best to move such changes unrelated to the main goal of the PR to separate PRs. Let the reviewers focus on what really matters by minimising the noise.

Reply:
@snowleopard I'll revert it for now since its not important.